source/application/AMPIC887DriverServer/AMGCS2.h

```diff
@@ -0,0 +1,107 @@
+#ifndef AMGCS2_H
+#define AMGCS2_H
+
+#include <QChar>
+/*!
+  * Namespace which contains all the enumerators for use with the GCS2.0 syntax
+  * provided by the PI C887.11 controllers.
+  */
+namespace AMGCS2 {
+
+	/***************
+	  * Enums
+	  ****************************/
+
+	/*!
+	  * Enumerates all the hexapod axes controlled by a C887.11 controller.
+	  */
+	enum Axis {
+		UnknownAxis = -1,
+		XAxis = 0,
```

Comment:
Although more of an aesthetic comment, you probably don't need to be as pedantic since the enum is `Axis`.  Inside of the `Axis` enum you could just have `X, Y, Z, U, V, W` rather than `XAxis`.  What do you think?